.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
It would be helpful to provide people with an example of what we consider a "clear and concise" description.

Reply:
Ah, we don't have a good example yet... Let me make up one and you can help to make sure they are truly clear and concise.